tasks/sanity-inspect-image.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-inspect-image
+spec:
+  params:
+    - name: SOURCE_IMAGE
+      description: the fully qualified image name
+      default: quay.io/hongweiliu/skopeo-inspect-test:latest
+      type: string
+    - name: BASE
+      description: is to get base image?
```

Comment:
Maybe change description to something like: **Represents base image**

Reply:
Thank you, how about changing to `Inspecting base image for source image`?

Reply:
I think that describing variable with process that happens to it is not necessary. What do you think?

Reply:
Yes. You are right.
And since the `base image` should exist by default, so I revise the code and will get the the manifest for the source image and base image at the same time without the param. Could you take a look?